### DISCUSSION TITLE
How to recover deleted github Branch

### DISCUSSION THREAD

**Discussion by karan-syan** [community/community#44699] [New to GitHub]
### Select Topic Area

Product Feedback

### Body

How to recover a deleted branch in github i researched a lot but there isn't any solution regarding it

**Comment by SilasRodrigues19** [comment]
Hey @karan-syan 

Maybe these links will help you.

https://gist.github.com/umayr/b95e11d5f22c24a872ef95d215ba2ab1
https://gist.github.com/jbgo/1944238

**Comment by GabyCT** [comment]
Here it is some docs about that
https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/deleting-and-restoring-branches-in-a-pull-request#restoring-a-deleted-branch

**Comment by radar-blip** [comment]
You could try here: [Support Page](https://support.github.com/request) under `Restore deleted data`

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @karan-syan ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by Marcono1234** [comment]
Can be done in the Activity tab of the repository, see [this answer](https://github.com/orgs/community/discussions/55884#discussioncomment-9331142).

**Comment by nabielnx** [comment]
use git reflog
